GTG/gtk/editor/editor.py

```diff
@@ -209,10 +209,8 @@ def __init__(self, app, task):
             if sub.id not in self.textview.subtasks['tags']:
                 self.textview.insert_existing_subtask(sub)
 
-        # if thisisnew:
-        #     self.textview.select_title()
-        # else:
-            # self.task.set_to_keep()
```

Comment:
I'm not sure what this was doing, and since the method no longer exists, I figured we probably don't need to keep this line around.

Reply:
Definitely, it was related to liblarch's way of managing things. I think it would delete the task if it didn't have the else block? I also can't remember now